desktop/src/main/java/bisq/desktop/main/support/dispute/DisputeView.java

```diff
@@ -542,6 +544,11 @@ protected boolean reOpenDispute() {
     private boolean isDisputeWithSameCurrentNodeAddress(Dispute dispute, boolean isMediator) {
         NodeAddress disputeNodeAddress = isMediator ? dispute.getContract().getMediatorNodeAddress() :
                 dispute.getContract().getMyNodeAddress(keyRing.getPubKeyRing());
+
+        if (OfferRestrictions.requiresNodeAddressUpdate() && !Utils.isV3Address(disputeNodeAddress.getHostName())) {
+            return false;
+        }
```

Comment:
This doesn't really fit within the scope of the name of this method since it's failing same addresses if they're V2 after the deactivation date. Perhaps renaming this method to something more generic like `hasOkAddress()` to catch all the conditions.